libs.versions.toml

```diff
@@ -175,7 +176,7 @@ leakcanary-android = { module = "com.squareup.leakcanary:leakcanary-android", ve
 mockito-android = { module = "org.mockito:mockito-android", version.ref = "mockito" }
 mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
 mockito-inline = { module = "org.mockito:mockito-inline", version.ref = "mockito" }
-mockito-kotlin = { module = "org.mockito.kotlin:mockito-kotlin", version.ref = "mockito" }
+mockito-kotlin = { module = "org.mockito.kotlin:mockito-kotlin", version.ref = "mockito-kotlin" }
```

Comment:
`mockito-core` and `mockito-kotlin` dependencies are not in sync. See:
- [`mockito-kotlin` releases](https://github.com/mockito/mockito-kotlin/releases)
- [`mockito-core` releases](https://github.com/mockito/mockito/releases)

Syncing those versions leads to incompatibility issues.

To reproduce, remove `version("mockito", "3.3.3")` line in FluxC. This will force `mockito` and `mockito-kotlin` to be in version `4.0.0`. Then, run unit tests and observe that `mockito-kotlin` references to nonexisting method `Mockito#verifyZeroInteractions` in `mockito-core`.